zegami_sdk/util.py

```diff
@@ -175,7 +175,7 @@ def _auth_put(self, url, body, return_response=False, **kwargs):
         url, body, verify=not ALLOW_INSECURE_SSL, **kwargs
     )
     self._check_status(r, is_async_request=False)
-    return r if return_response else r.json()
+    return r if return_response
```

Comment:
Sometimes we do want the json. I think we can check the response code, if `204` just return, otherwise return `r.json()`